src/main/java/dog/pawbook/model/managedentity/owner/Owner.java

```diff
@@ -24,7 +25,7 @@
 
     // Data fields
     private final Address address;
-
+    private final ArrayList<Integer> dogIdList;
```

Comment:
arrayset might be better?

Reply:
Hmm then maybe HashSet will do also, btw can just leave it without the specific type of implementation, i.e. `List` instead of `ArrayList`, `Set` instead of `Hashset`, there's a discussion on the forum on this.

Reply:
and a set makes more sense for our case, since we don't really need an order to them, it is more ideal in the sense, but why not get it right at the start and not worry later.